examples/output_voltage_measurement/README.md

```diff
@@ -0,0 +1,44 @@
+## Output Voltage Measurement
+
+This is a MeasurementLink example that sources DC voltage as input with an NI SMU and measures output using NI-VISA DMM.
```

Comment:
"NI-VISA DMM" may be useful shorthand but it's imprecise at best and misleading at worst. That's why the other readme doesn't use this term.

Reply:
Updated to `... measures the DUT output with DMM using NI-VISA.`.